packages/utils/src/RBAC/RBAC.ts

```diff
@@ -60,4 +61,20 @@ export function hasAllPermissions(userPermissions: (Access | string)[], permissi
   });
 }
 
+export const initialPermissions = {
+  isLoading: true,
+  isOrgAdmin: false,
+  permissions: [],
+  hasAccess: (_?: any) => {
```

Comment:
the `hasAccess` argument should have the actual types defined `(Access | string)[]`. Based on the values used at runtime.

Reply:
Update.